consensus/src/quorum_store/proof_manager.rs

```diff
@@ -84,8 +87,11 @@ impl ProofManager {
         self.batch_proof_queue.mark_committed(batches);
         self.batch_proof_queue
             .handle_updated_block_timestamp(block_timestamp);
-        (self.remaining_total_txn_num, self.remaining_total_proof_num) =
-            self.batch_proof_queue.remaining_txns_and_proofs();
+        sample!(
```

Comment:
Should we wrap this into a function `update_remaining_txns_and_proofs` and have the SampleRate in that function?

Reply:
Done.